src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+
+    Order add(Order order);
+
+    Order getbyUser(User user);

```

Comment:
The method name `getbyUser` should be changed to `getByUser` to follow the camelCase naming convention.

Reply:
ok, done